### DISCUSSION TITLE
Tech stack and project structure

### DISCUSSION THREAD

**Discussion by UsamaKhan** [karakeep-app/karakeep#142] [Q&A] [answered]
Hi,

I am looking at creating a personal project and this app caught my eye when it comes to how the project is structured.

I know the [readme](https://github.com/MohamedBassem/hoarder-app?tab=readme-ov-file#stack) mentions the stack used but I don't see how the mobile end is set up.

Also, I wanted to know if there was a specific way you scaffolded this project? Did you use something like [solito](https://solito.dev/) or a template like [acme-corp](https://github.com/juliusmarminge/acme-corp)? Is there something that you would recommend for setting up a project for both web and mobile?

**Comment by MohamedBassem** [comment, marked as answer]
Hi, I'm glad that this app caught your eye!

I didn't use templates to scaffold the project. However, the stack turned out to be very similar to that of [T3 app](https://create.t3.gg/), so I ended up checking their templates and copying some of the stuff from there.

In particular this repo (https://github.com/t3-oss/create-t3-turbo) which I ended up following their structure and tooling. So if you're asking for my advice, go ahead and use this template.

**Reply by UsamaKhan** [reply]
Thanks for they reply. I'll take a look at the t3 template!
